app/services/github_service.rb

```diff
@@ -17,14 +17,13 @@ def followings
   end
 
   def get_user(username)
-    get_json("/users/#{username}")
+    get_json("/users/#{username}?access_token=#{@token}")
```

Comment:
After re-reading github api docs, the user does need an access token to see publicly listed emails, so added this and checked it in local host with dream catcher and everything is working great